Server/sprawl_calculator/wup_calculator/views.py

```diff
@@ -1,13 +1,74 @@
 from django.shortcuts import render
 from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+from datetime import datetime
+from dataclasses import asdict
+from .models import WUPCalculation
+from django.http import JsonResponse
+from .data_definations import WUPCalculationResult, WUPCalculationRequestActivating, WUPCalculationRequestProcessing, WUPCalculationRequestCompleted, WUPCalculationRequestRejected, WUPCalculationRequestError, WUPIndexGeneratorRequest
 # Create your views here.
 
-#Endpoint1_View
+
+""" Start a WUP index calculation processing request """
+
 @api_view(['PUT', 'GET'])
```

Comment:
Let's remove GET (per the documentation)

Reply:
GET method removed from [the endpoint](roshanborkar01:GET_Removed).